src/itemfinder.c

```diff
@@ -0,0 +1,659 @@
+#include "global.h"
+#include "task.h"
+#include "new_menu_helpers.h"
+#include "strings.h"
+#include "event_scripts.h"
+#include "map_obj_lock.h"
+#include "script.h"
+#include "sound.h"
+#include "event_data.h"
+#include "field_player_avatar.h"
+#include "field_specials.h"
+#include "fieldmap.h"
+#include "itemfinder.h"
+#include "constants/songs.h"
+
+static void Task_NoResponse_CleanUp(u8 taskId);
+static void Task_ItemfinderResponseSoundsAndAnims(u8 taskId);
+static void Task_ItemfinderUnderfootSoundsAndAnims(u8 taskId);
+static bool8 HiddenItemIsWithinRangeOfPlayer(struct MapEvents *events, u8 taskId);
```

Comment:
Again inconsistent pointer space style through this file

Reply:
Fixed